modules/meeting/app/components/meeting_agenda_items/item_component/show_component.sass

```
@@ -22,13 +26,21 @@ $meeting-agenda-item--presenter-width: 300px
 
   &--duration
     white-space: nowrap
+    justify-self: end
+
+  &--title
+    overflow-wrap: anywhere
+    margin-bottom: 0
 
   &--content
-    @include text-shortener
+    overflow-wrap: anywhere
 
   @media screen and (max-width: $breakpoint-lg)
-    grid-template-columns: 20px auto 1fr calc($meeting-agenda-item--presenter-width - 50px) 50px
-    grid-template-areas: "drag-handle content content content actions" ". duration . presenter presenter" ". notes notes notes notes"
+    grid-template-columns: 20px auto 1fr 150px 50px
```

Why don't you use the variable you introduced?
```suggestion
    grid-template-columns: 20px auto 1fr $meeting-agenda-item--presenter-width--mobile 50px
```